src/pages/Home/Menu/CategoryLi.style.ts

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+
+export const CategoryLiStyle = styled.li`
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  list-style: none;
+  width: 91.3%;
+  height: 3.6875rem;
+  margin: auto;
+  border-bottom: 0.0625rem solid #e8e8e8;
+  color: ${(props) => props.color};
```

Comment:
you should not be passing the color to the element. Do it like this


interface Props {
$selected: boolean

export const CategoryLiStyle = styled.li<Props>\`
...
color: ${({$selected}) => selected ?  "#color1" : "#color2" } 
\`

Reply:
The $ in front of selected is to prevent the selected variable from being passed to the html element itself